### ISSUE TITLE
Removing newlines in feature product description doesn't add a space.

### ISSUE THREAD

**Issue by Koeng101** [TimothyStiles/poly#8]
**Describe the bug**
When parsing out descriptions, newlines aren't replaced with a space. Newlines should be replaced with a space and not with nothing.

Genbank:
```
     CDS             410..1750
                     /gene="dnaA"
                     /locus_tag="BSU_00010"
                     /old_locus_tag="BSU00010"
                     /function="16.9: Replicate"
                     /experiment="publication(s) with functional evidences,
                     PMID:2167836, 2846289, 12682299, 16120674, 1779750,
                     28166228"
                     /note="Evidence 1a: Function from experimental evidences
                     in the studied strain; PubMedId: 2167836, 2846289,
                     12682299, 16120674, 1779750, 28166228; Product type f :
                     factor"
                     /codon_start=1
                     /transl_table=11
                     /product="chromosomal replication initiator informational
                     ATPase"
                     /protein_id="NP_387882.1"
```
JSON:
```
    "product": "chromosomal replication initiator informationalATPase",
```

**To Reproduce**
`cat bsub.gbk | poly c -i gbk -o json > bsub.json`
Check `bsub.gbk` and `bsub.json` with your favorite text editor.

**Expected behavior**
I expect there to be a space

**Screenshots**

**Desktop (please complete the following information):**
 - Arch linux with go

**Additional context**

**Comment by TimothyStiles** [comment]
Oh man. I labeled this as a good first issue but for some reason what I thought was the correct quick fix is breaking the convert command test. I'll play around with it and update in a bit.

**Comment by TimothyStiles** [comment]
Double oof. There's nothing differentiating between what needs a space and what doesn't. The cheapest fix to this would be to add a space except for some different qualifiers like translation like translation this would break the sequence.

I could just flag for translation and any other official sequence qualifiers that may exist but still. Yeesh.

**Comment by TimothyStiles** [comment]
I just fixed this with my latest push 25abfb5. Everything should work now as expected. Take a look and let me know what you think!
